projects/packages/codesniffer/README.md

```diff
@@ -28,7 +28,7 @@ You should then include the Jetpack rules in your `.phpcs.xml.dist`, like
 You will also likely want to set some configuration for other included rulesets:
 ```xml
 <config name="minimum_supported_wp_version" value="5.4" />
```

Comment:
Should probably bump this to at least 6.3 too.

Reply:
Done in 51746059f9565019d372d6bcfe6dce647076ea8e